hosting/src/store/index.ts

```diff
@@ -15,7 +16,13 @@ const store = createStore<PTVuexState>({
     currentUser: null,
     events: []
   },
-  getters: {},
+  getters: {
+    events: (state) => state.events.map(eventToEventUi),
```

Comment:
Re: our discussion on reusing `events` being confusing, do we ever want the raw Firestore version? Perhaps we should rename the state field to `futureEventsRaw` and call the getter `futureEvents`.

Reply:
DONE - renamed state field to `eventsRaw`